src/Modules/Workspaces/Commands.Workspaces/help/Add-PowerBIWorkspaceUser.md

```diff
@@ -0,0 +1,106 @@
+---
+external help file: Microsoft.PowerBI.Commands.Workspaces.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Workspaces
+online version:
+schema: 2.0.0
+---
+
+# Add-PowerBIWorkspaceUser
+
+## SYNOPSIS
+Gives permissions to a given user to a Power BI workspace 
+
+## SYNTAX
+
+```
+Add-PowerBIWorkspaceUser [-Scope <PowerBIUserScope>] -Id <Guid> -UserEmailAddress <String>
+ -UserAccessRight <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Add-PowerBIWorkspaceUser -Scope Organization -GroupId 23FCBDBD-A979-45D8-B1C8-6D21E0F4BE50 -UserEmailAddress john@contoso.com
+ -UserAccessRight Admin
```

Comment:
This will need to be on line 26 too or use ` to indicate the statement runs to the next line

Reply:
Resolved.